src/cli.js

```diff
@@ -4,12 +4,16 @@ const main = require('./main.js');
 const pkg = require('../package.json');
 const chalk = require('chalk');
 const cachedir = require('cachedir');
+require('dotenv').config()
 
 const help = `
 Usage: starred_search [OPTIONS] [ARGS]...
 
   Search your or any other user's starred repositories on GitHub for a keyword.
 
+  Reads from $GITHUB_TOKEN env variable an oprtional github access token to be used in sending requests.
+  Generate one from here https://github.com/settings/tokens?type=beta
```

Comment:
```suggestion
```

Let's skip it here, we'll add it to the README

Reply:
Okay removed from the `-h` arg, how you want it shown in the readme?